hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh

```diff
@@ -213,16 +213,18 @@ log::info "Checking if the ES cluster is all yellow/green"
 try_func_until_text_alt get_es_cluster_status "\"green\"" "\"yellow\"" ${ES_POD_TIMEOUT}
 
 # read new 4.5 indices and map them by their names
-log::info "Reading new ES indices"
-try_func_until_result_is_not_empty get_es_indices_names ${ES_POD_TIMEOUT}
-new_indices=$(get_es_indices_names)
-
-if [ "$old_indices" != "$new_indices" ]; then
-  log::info "Test failed - Indices names not matched"
-  exit 1
-fi
-
-log::info "Retrieving record inserted before migration"
+#log::info "Reading new ES indices"
+#try_func_until_result_is_not_empty get_es_indices ${ES_POD_TIMEOUT}
+#new_indices=$(get_es_indices_names)
+
+#if [ "$old_indices" != "$new_indices" ]; then
+  #log::info "Test failed - Indices names not matched"
+  #log::info "Old indices: $old_indices"
+  #log::info "New indices: $new_indices"
+  #exit 1
+#fi
```

Comment:
What is the intention here by commenting out? Testing or CI or permanently disable this checks? If latter applies I suggest to delete entirely.

Reply:
I'd keep that so when we move this test to the golang we won't forget about verifying all indices

Reply:
@igor-karpukhin Please ensure this happens also on master please.